### PR TITLE
[Intel MKL] Add MatMul+BiasAdd+Add fusion

### PR DESCRIPTION
**Pull request by i8run**
This PR add a new fusion pattern `MatMul + BiasAdd + Add`, which reuse some code of `Conv2D + BiasAdd + Add` because of similar pattern.

For tests, it adds two, one for remapper and the other for fused results.

Signed-off-by: Wang Yanzhang yanzhang.wang@intel.com

### CODE REVIEW

**Comment by i8run** [comment]
@googlebot I signed it!

**Comment by i8run** [comment]
@googlebot I consent.

File: tensorflow/core/grappler/optimizers/remapper.cc [hunk 1]
```diff
@@ -1283,28 +1283,34 @@ Status AddFusedContractionNode(RemapperContext* ctx,
...
   const NodeDef& add = graph->node(matched.add);
-  fused_conv2d.set_name(add.name());
-  fused_conv2d.set_op(kFusedConv2D);
-  fused_conv2d.set_device(contraction.device());
-  fused_conv2d.add_input(contraction.input(0));  // 0: input
-  fused_conv2d.add_input(contraction.input(1));  // 1: filter
-  fused_conv2d.add_input(bias_add.input(1));     // 2: bias
+  contraction_node.set_name(add.name());
+  contraction_node.set_device(contraction.device());
+  contraction_node.add_input(contraction.input(0));  // 0: input
```
Comment (ezhulenev) [inline tensorflow/core/grappler/optimizers/remapper.cc:1293]: `0: input` and `1: filter` is a bit misleading for matched MatMul

**Review by ezhulenev — CHANGES_REQUESTED** [review]

File: tensorflow/core/grappler/optimizers/remapper.cc (hunk 1 above)
Comment (i8run) [reply tensorflow/core/grappler/optimizers/remapper.cc]: Modified. Done. Thanks @ezhulenev .

**Review by ezhulenev — APPROVED** [review]

**Pull request merged**